readme.md

```diff
@@ -53,7 +81,7 @@ console.log(results);
 ```
 
 Your Answer:
-```text
+```I think you just need '.findById'
```

Comment:
Not quite. What we were looking for here was the fact that `.find` (and all other Mongoose DB methods) are asynchronous, and thus you have to give them a callback function (or use `.done()`) to tell them what to do once the results come in.